### PR TITLE
added correct link

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #8370, so i added correct link .
**https://docs.microsoft.com/microsoft-365/security/defender-endpoint/non-windows?view=o365-worldwide**
above link is working

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Defender for Endpoint uses the following combination of technology built into Wi
 
 
 >[!IMPORTANT]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Defender for Endpoint uses the following combination of technology built into Wi
...
->The capabilities on non-Windows platforms may be different from the ones for Windows. For more information on what capabilities are available for non-Windows platforms, see [Microsoft Defender for Endpoint for non-Windows platforms](/security/defender-endpoint/non-windows).
+>The capabilities on non-Windows platforms may be different from the ones for Windows. For more information on what capabilities are available for non-Windows platforms, see [Microsoft Defender for Endpoint for non-Windows platforms](https://docs.microsoft.com/microsoft-365/security/defender-endpoint/non-windows?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint.md]:
```suggestion
> The capabilities on non-Windows platforms may be different from the ones for Windows. For more information on what capabilities are available for non-Windows platforms, see [Microsoft Defender for Endpoint for non-Windows platforms](/microsoft-365/security/defender-endpoint/non-windows).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**